policyengine_us/parameters/gov/states/co/tax/income/credits/income_qualified_senior_housing/credit_threshold_separate.yaml

```diff
@@ -0,0 +1,431 @@
+description: CO Income Qualified Senior Housing Income Tax Credit threshold for single, joint and head of household filing status.
+metadata:
+  unit: currency-USD
+  type: single_amount
+  label: CO Income Qualified Senior Housing Income Tax Credit threshold for single, joint and hoh
+  period: year
+  reference:
+    - title: Income Qualified Senior Housing Income Tax Credit
+      href: https://tax.colorado.gov/income-qualified-senior-housing-income-tax-credit
+    - title: 2022 Colorado Individual Income Tax Filing Guide
+      href: https://tax.colorado.gov/sites/tax/files/documents/DR_104_Book_2022.pdf#page=17
+
+# def printall(left_bound, initial_credit): # threshold printer function
```

Comment:
delete the commented lines

Reply:
This file will be deleted later since I built "reduction".